src/workerd/jsg/jsg.c++

```diff
@@ -278,27 +278,68 @@ JsSymbol Lock::symbolAsyncDispose() {
   return IsolateBase::from(v8Isolate).getSymbolAsyncDispose();
 }
 
-void Lock::adjustExternalMemory(ssize_t amount) {
-  v8Isolate->AdjustAmountOfExternalAllocatedMemory(static_cast<int64_t>(amount));
+void ExternalMemoryAdjuster::maybeDeferAdjustment(v8::Isolate* isolate, size_t amount) {
```

Comment:
Minor, ignore if you want, but I found it slightly confusing that this represents an inverted adjustment, and we talk about deferring a "decrement"... if we just made this `ssize_t` and called it a "deferred adjustment" (which always happens to be negative in practice since it's used by the destructor) it seems clearer.

Reply:
Will leave it as is for now, can revisit this later if it proves helpful to do so